packages/metadata/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@sap-ux/metadata",
+  "version": "0.1.0",
+  "description": "Metadata Service",
+  "main": "dist/index.js",
+  "author": "SAP SE",
+  "license": "Apache-2.0",
+  "private": false,
+  "scripts": {
+    "build": "npm-run-all clean && tsc --build ./tsconfig.build.json",
+    "clean:dist": "rimraf ./dist ./generators *.tsbuildinfo",
+    "clean": "rimraf ./reports",
+    "format:fix": "prettier --write --loglevel silent --ignore-path ../../../.prettierignore",
+    "format:fix:all": "prettier --write '**/*.{css,scss,html,js,json,ts,tsx,yaml,yml}' '!**/{out,dist,typings,node_modules}/**' '!**/*.{svg,png,xml}' --ignore-path ../../../.prettierignore",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:summary": "eslint . --ext .ts,.tsx -f summary",
+    "lint:fix": "eslint --fix",
+    "lint:fix:all": "eslint . --ext .ts,.tsx --fix",
+    "lint:report": "eslint . --ext .ts,.tsx -f multiple ",
+    "madge": "madge --warning --circular --extensions ts ./",
+    "pre-commit": "lint-staged --quiet",
+    "test": "jest  --maxWorkers=1 --ci --forceExit --detectOpenHandles",
+    "updateVocabularies": "npm-run-all build && node ./bin/UpdateVocabularies.js",
```

Comment:
This does not exist and should be removed.

Reply:
removed